src/core/qgsdistancearea.h

```diff
@@ -304,6 +319,25 @@ class CORE_EXPORT QgsDistanceArea
     double computeDistanceBearing( const QgsPoint &p1, const QgsPoint &p2,
                                    double *course1 = nullptr, double *course2 = nullptr ) const;
 
+    /**
+     * Given a location, an azimuth and a distance, computes the
+     * location of the projected point. Based on Vincenty's formula
+     * for the geodetic direct problem as described in "Geocentric
+     * Datum of Australia Technical Manual", Chapter 4. Tested against:
```

Comment:
Can you make everything in this paragraph after "chapter 4" and up to the \since a normal c++ comment and not include it in the doxygen API notes. There's too much detail here which is only of use for working on the method, and not for using it.